### DISCUSSION TITLE
How to Write Bufr？

### DISCUSSION THREAD

**Discussion by zfy68** [Unidata/netcdf-java#1024] [Questions and Answers] [answered]
**Is there an implementation class for writing bufr?**
I want to modify the bufr value based on the existing bufr file。

**Comment by JohnLCaron** [comment, marked as answer]
We dont support writing to bufr.

**Reply by zfy68** [reply]
oh，thanks

**Comment by shahramn** [comment]
You can use ecCodes ( see https://confluence.ecmwf.int/display/ECC )

**Reply by zfy68** [reply]
thanks. I already looked.I just want to encode bufr by Java code.
